ACPs/194-streaming-asynchronous-execution/README.md

```diff
@@ -118,49 +125,123 @@ ACP-103 introduced the following variables for calculating the gas price:
 
 </div>
 
-ACP-176 set $R = 2T$ and provided a mechanism to make $T$ dynamic.
+ACP-176 provided a mechanism to make $T$ dynamic and set:
+
+$$
+\begin{align}
+R &= 2 \cdot T \\
+K &= 87 \cdot T
+\end{align}
+$$
+
 The _excess_ actual consumption $x \ge 0$ beyond the target $T$ is tracked via numerical integration and used to calculate the gas price as:
 
 $$M \cdot \exp\left(\frac{x}{K}\right)$$
 
-While ACPs 103 and 176 compute this price at the resolution of a single block, we can instead integrate at the resolution of a single transaction with equivalent pricing dynamics.
-Exploiting the proportionality of $R = pT$ with $p \ge 1$, for a transaction consuming $g$ gas, the excess is increased:
+### Gas charged
+
+We introduce $g_L$, $g_U$, and $g_C$ as the gas _limit_, _used_, and _charged_ per transaction, respectively. We define
+
+$$
+g_C := \max\left(g_U, \frac{g_L}{\lambda}\right)
+$$
+
+where $\lambda$ enforces a lower bound on the gas charged based on the gas limit.
```

Comment:
what is lambda? a constant or a dynamic value?

Reply:
It is a constant. At the end of the ACP it is suggested to be set equal to 2.